docs/content/en/docs/getting-started/observability/_index.md

```diff
@@ -1,280 +1,216 @@
 
-## DORA metrics
+## Enable KLT for your cluster
 
-DORA metrics are an industry-standard set of measurements;
-see the following for a description:
+To enable KLT for your cluster, annotate the
+[Namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/)
+resource
+In this example, this is defined in the
+[simplenode-dev-ns.yaml](https://github.com/keptn-sandbox/klt-on-k3s-with-argocd/blob/main/simplenode-dev/simplenode-dev-ns.yaml)
+file, which looks like this:
 
-- [What are DORA Metrics and Why Do They Matter?](https://codeclimate.com/blog/dora-metrics)
-- [Are you an Elite DevOps Performer?
-   Find out with the Four Keys Project](https://cloud.google.com/blog/products/devops-sre/using-the-four-keys-to-measure-your-devops-performance)
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: simplenode-dev
+  annotations:
+    keptn.sh/lifecycle-toolkit: "enabled"
+```
 
-DORA metrics provide information such as:
+You see the annotation line that enables `lifecycle-toolkit`.
+This line tells the webhook to handle the namespace
 
-- How many deployments happened in the last six hours?
-- Time between deployments
-- Deployment time between versions
-- Average time between versions.
+## Integrate KLT with your cluster
 
-The Keptn Lifecycle Toolkit starts collecting these metrics
-as soon as you annotate the `Deployment` resource.
-Metrics are collected only for the `Deployment` resources
-that are annotated.
+To integrate KLT with your cluster, annotate the Kubernetes
+[Deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/)
+resource.
+In this example, this is defined in the
+[simplenode-dev-deployment.yaml](https://github.com/keptn-sandbox/klt-on-k3s-with-argocd/blob/main/simplenode-dev/simplenode-dev-deployment.yaml)
+file, which includes the following lines:
 
-To view DORA metrics, run the following command:
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: simplenode
+  namespace: simplenode-dev
+...
+template:
+    metadata:
+      labels:
+        app: simplenode
+        app.kubernetes.io/name: simplenodeservice
+      annotations:
+        # keptn.sh/app: simpleapp
+        keptn.sh/workload: simplenode
+        keptn.sh/version: 1.0.2
+        keptn.sh/pre-deployment-evaluations: evaluate-dependencies
+        keptn.sh/pre-deployment-tasks: notify
+        keptn.sh/post-deployment-evaluations: evaluate-deployment
+        keptn.sh/post-deployment-tasks: notify
+...
+```
 
-```shell
-kubectl port-forward -n keptn-lifecycle-toolkit-system \
-   svc/lifecycle-operator-metrics-service 2222
+For more information about using annotations and labels
+to integrate KLT into your deployment cluster, see
+[Integrate KLT with your applications](../../implementing/integrate/_index.md).
+
+## Define evaluations to be performed pre- and post-deployment
+
+An `evaluation` is a KeptnMetric that has a defined target value.
```

Comment:
an evaluation is not a keptn metric and I would differentiate between Evaluation and Evaluation Definitions here they seem mixed up... Maybe something like: a user can configure evaluation definitions to associate a desired target to a KeptainMetric. These evaluation definitions are then referred by name in a workload or app, to run at pre- or post-deployment. KLT will then create an evaluation for each workload or app version referring to the evaluation definition and store the result of whether or not the Keptnmetric satisfies the target at check time, inside the evaluation resource.

Reply:
not sure this is any clearer than before, but maybe it can help :D